source/js/src/utils.js

```diff
@@ -265,7 +266,7 @@ NexT.utils = NexT.$u = {
   },
 
   getContentVisibilityHeight: function() {
-    var docHeight = $('#content').height();
+    var docHeight = $('.container').height();
```

Comment:
Container again? Please check back2top when comments will be:

1. Offed.
2. Lazy loaded (disqus, for e.x.)
3. Not lazy loaded.

Also, mb better will be use `#main` instead of `.container`? ID,s faster then classes.

Reply:
I have considered it, and seems `$('.container')` or `$('body')` is the best choice. `$('#content')` doesn't calculate the height of comment system, and `$('#main')` doesn't include the footer. Besides, `getContentVisibilityHeight` is executed each time the page scrolls, so `docHeight` will always be refreshed even if some content is lazy-loaded.